htmd/parameterization/fftype.py

```diff
@@ -289,79 +278,31 @@ def test_mol(self):
 
         for name, method, chargetuple in toTest:
             with self.subTest(name=name, method=method, chargetuple=chargetuple):
+
                 refDir = os.path.join(self.refDir, name, method)
                 self._init_mol(name, method, chargetuple)
-                # self._generate_references(name, method)
```

Comment:
This is good to keep to remember people they can generate the references with this function

Reply:
OK! This can stay.